### PR TITLE
[CCXDEV-10678] Add feature for aggregator mock and upgrades responses

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Add feature for aggregator mock and upgrades responses

Part of [CCXDEV-10678](https://issues.redhat.com/browse/CCXDEV-10678)

## Type of change

- New test feature file

## Testing steps

Locally.

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container

### CODE REVIEW

File: features/insights-results-aggregator-mock/upgrades.feature [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+                "type": "object",
+                "properties": {
+                    "last_checked_at": {
+                    "type": "string",
+                    "format": "date"
+                    }
+                }
+                }
+            }
+            }
```
Comment (tisnik) [inline features/insights-results-aggregator-mock/upgrades.feature:84]: is the formatting off?

File: features/insights-results-aggregator-mock/upgrades.feature [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
```
Comment (tisnik) [inline features/insights-results-aggregator-mock/upgrades.feature:162]: dtto

**Review by tisnik — APPROVED** [review]
oukey

File: docs/scenarios_list.md [hunk 3]
```diff
@@ -754,6 +754,15 @@ nav_order: 3
 * Check if Insights Results Aggregator Mock displays authors
 * Check if Insights Results Aggregator Mock displays actual configuration
 
+## `insights-results-aggregator-mock/upgrades.feature`
+
+* Check if Insights Results Aggregator Mock service return upgrade recommended for a cluster without risks
```
Comment (epapbak) [inline docs/scenarios_list.md:759]: returns

File: features/insights-results-aggregator-mock/upgrades.feature [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+  Background:
+    Given REST API service hostname is localhost
+      And REST API service port is 8080
+      And REST API service prefix is /api/insights-results-aggregator/v2
+
+
+  @rest-api
+  Scenario: Check if Insights Results Aggregator Mock service return upgrade recommended for a cluster without risks
+    Given the system is in default state
```
Comment (epapbak) [inline features/insights-results-aggregator-mock/upgrades.feature:11]: Could be added to background

**Review by epapbak — COMMENTED** [review]
small comments, but +1 overall

**Review by epapbak — APPROVED** [review]

**Pull request merged**